drivers/mm/mm_drv_intel_adsp_mtl_tlb.c

```diff
@@ -277,8 +297,12 @@ int sys_mm_drv_map_page(void *virt, uintptr_t phys, uint32_t flags)
 
 	entry_idx = get_tlb_entry_idx(va);
 
-	bank_idx = get_hpsram_bank_idx(pa);
+#ifdef CONFIG_SOC_INTEL_COMM_WIDGET
+	used_pages++;
```

Comment:
Why this is calculated?
PMC needs report about banks usage, in this code you already know when power on/off occurs, you can report to PMC in such cases

Reply:
The power supply controls 128KB blocks, the use of 32kb blocks is reported to the PMC.

Reply:
So this make sense now but it needs to be marked somewhere, in the same file we already have memory "banks" which means actually something else than newly added "banks".
At least please add comments but best to use different names everywhere where this is different bank, maybe pmc_bank? and comment to used_banks_reported what this means